lib/lotus/cli.rb

```diff
@@ -73,6 +73,7 @@ def routes
     end
 
     desc 'new', 'generates a new application'
+    method_option :adapter,                       desc: 'adapter config',             type: :string, default: 'filesystem'
```

Comment:
@Linuus I think `database` is better :+1: 

The problem with `adapter` is that a generic word and it has no direct link with the database world.